pipelines/build_pyrenew_model.py

```diff
@@ -75,7 +74,7 @@ def build_model_from_dir(
             "inf_to_hosp_admit",
             reference_loc=model_data["inf_to_hosp_admit_lognormal_loc"],
             reference_scale=model_data["inf_to_hosp_admit_lognormal_scale"],
-            n=jnp.size(model_data["inf_to_hosp_admit_pmf"]) * 2,
+            n=jnp.size(jnp.array(model_data["inf_to_hosp_admit_pmf"])) * 2,
```

Comment:
why is this needed?

Reply:
Deprecation warning when `model_data["inf_to_hosp_admit_pmf"]` was not `ArrayLike`

Reply:
In that case I think I'd do 
```suggestion
            n=jnp.array(model_data["inf_to_hosp_admit_pmf"]).size * 2,
```